index.html

```diff
@@ -0,0 +1,114 @@
+            </ul>                
+            <button class="hamburger-open"><img src="./img/Hamburger_icon.png" alt="hamburger-icon"></button>                
+        </nav> 
+    </header>
+
+    <main>
+        <section class="headline">            
+                <h2>"Hello hikers of the world!"</h2>
+                <h1>HIKING AND ADVENTURE SPORTS<br> SEASON 2023</h1>            
+                <p class="headline__p">Hiking and trekking · Ski and snowboard · Mountainbike · Horseback riding · Mountaineering and climbing · Surf · Sport fishing · Kayak <br> Are part of the adventure that awaits you this new season</p>
+                <h3>2023.04.18(thu) ~ 21(FRI)</h3>
+                <p>@Pucón, región de la Araucanía, Chile</p>            
+        </section>
+
+        <section class="program" id="program">            
+                <h2>Main Program</h2>
+                <hr>
+                <ul class="program__ul">
+                    <li class="program__ul--bike">
+                        <img src="./img/biking.svg" alt="biking">
+                        <h3>Mountain bike</h3>
+                        <p>The full suspension bicycle represents the most important advance in cycling in the 20th century; since it provides cyclists with the possibility of reaching places that in another time were considered inaccessible to said vehicle.</p>
+                    </li>
+                    <li>
+                        <img src="./img/ski.svg" alt="skiing">
+                        <h3>Skiing</h3>
+                        <p>Skiing is a mountain sport that consists of sliding through the snow, by means of two boards attached to the sole of the skier's boots by means of mechanical-robotic fixings, with multiple buttons with various functions.</p>
+                    </li>
+                    <li>
+                        <img src="./img/volcano.svg" alt="volcano">
+                        <h3>Villarrica Volcano</h3>
+                        <p>Visible from miles away, though if you spot a cheeky spurt, you may want to wait a few days until you try to conquer this monstrous, active volcano! The hike is around five hours, and is possible without prior training.</p>
+                    </li>
+                    <li>
+                        <img src="./img/water.svg" alt="water">
+                        <h3>Water sports</h3>
+                        <p>White-water rafting, Canyoning, kayaking, hydrospeed, are some of the water sports that you can enjoy.                      </p>
+                    </li>
+                    <li>
+                        <img src="./img/hiking.svg" alt="hiking">
+                        <h3>Hiking</h3>
+                        <p>Nestled on the shores of Lake Villarrica in Chile, Pucón is heaven for outdoor lovers. It is home to hot springs, active volcanoes, and several national parks. In this article, I will share my hiking and trekking experience in the Lake region of Chile.</p>
```

Comment:
At certain screen widths of the desktop version, the main program becomes two rows with 3 on top and two below making it different from the actual design. Kindly make sure all the cards stay in one row just as in the original version at all screen sizes of the desktop. 

### Your design 

![Screenshot from 2023-04-20 10-04-36](https://user-images.githubusercontent.com/65233860/233333434-546ed3e6-fc13-433e-b87b-7c8aa7172bd0.png)


### Actual design 

![Screenshot from 2023-04-20 09-57-27](https://user-images.githubusercontent.com/65233860/233333460-246ceeff-cdf8-4139-833f-b48f95d212b4.png)